node-operators/appchain-node/rpc-systemd.md

```diff
@@ -0,0 +1,265 @@
+
+To do so, take the following steps:
+
+1. Select `dataPreservers` as the module to query
+2. Set the storage query to `bootNodes`
+3. Provide your Tanssi Appchain ID
+4. Click on the **+** sign
+
+![Getting the bootnode](/images/node-operators/appchain-node/rpc-systemd/rpc-systemd-2.webp)
+
+### Run Flags {: #run-flags }
+
+The flags used in the `ExecStart` command can be adjusted according to your preferences and hardware configuration. The following ones are some of the most note-worthy:
+
+- `--name INSERT_NAME` - a human-readable name for this node
+- `--rpc-port INSERT_PORT` - specifies the JSON-RPC TCP port the node listens on
+- `--unsafe-rpc-external` - exposes the RPC service on all the interfaces
+- `--state-pruning INSERT_STATE_PRUNING_TYPE` - specifies when the Tanssi Appchain state should be removed from the database. The pruning type can be either `archive` (which makes the node behave as a full node keeping all the state), `archive-canonical` (which keeps only the state of finalized blocks), or any `number` (representing the number of blocks whose states are kept)
+- `--blocks-pruning INSERT_BLOCKS_PRUNING_TYPE` - specifies how many blocks should be kept in the database. The pruning type can be either `archive` (which makes the node behave as a full node keeping all the blocks), `archive-canonical` (which keeps only finalized blocks), or any `number` (representing the amount of finalized blocks to keep)
+- `--detailed-log-output` - enables detailed log output
+
+For a complete list of available flags, their description, and possible values, run the following command:
+
+=== "EVM-compatible Appchain"
+
+    ```bash
+    /var/lib/appchain-data/container-chain-template-frontier-node --help
+    ```
+
+=== "Simple Substrate Appchain"
+
+    ```bash
+    /var/lib/appchain-data/container-chain-template-simple-node --help
+    ```
+
+### Full Node Configuration Example for the Demo EVM Appchain {: #example-demo-evm-appchain}
+
+The following example is a fully functional full-node configuration for the [demo EVM Appchain](/builders/tanssi-network/networks/dancebox/demo-evm-containerchain/){target=\_blank} deployed on Dancebox with an ID of `3001`. 
+
+The raw chain specification file for the demo Appchain is required to run the node, and can be downloaded from this [public GitHub repository](https://github.com/papermoonio/external-files/blob/main/Tanssi/Demo-EVM-Appchain/){target=\_blank}. Download the file and place it in the `/var/lib/appchain-data/` directory.
+
+```bash
+[Unit]
+Description="Appchain systemd service"
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartSec=10
+User=appchain_node_service
+SyslogIdentifier=appchain
+SyslogFacility=local7
+KillSignal=SIGHUP
+ExecStart=/var/lib/appchain-data/container-chain-template-frontier-node \
+--chain=/var/lib/appchain-data/container-3001-raw-specs.json \
```

Comment:
similar comment as before: use the same structure and order of flags for docker. In fact, I just realized that the Demo EVM example is not following the same structure than the `Start-Up Command` section, can you fix that plz?